esp/templates/users/profiles/usercontact.html

```diff
@@ -1,6 +1,6 @@
 <!-- User ContactInfo -->
 
-<h3>Contact Information</h3>
+<h3>{{ profiletype|capfirst }} Contact Information</h3>
```

Comment:
Now that we've included this (which I think is a good change), will users be confused when the next section says "Teacher/Student/Guardian/Educator Information"? Should we maybe change those templates' headers?